joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include <chrono>
 #include <functional>
+#include <iostream>
```

Comment:
not needed ?

Reply:
yes, thanks. That was probably added automatically when I added some debugging at some point...